### PR TITLE
tags updated to 3.4.0 for q1

### PR DESCRIPTION
**Pull request by kgcarr**
Need to update image tags to the q1 release tag

### CODE REVIEW

**Comment by kgcarr** [comment]
IBM/ibm-mongodb-operator/issues/55

**Comment by chenzhiwei** [comment]
You also need to update the CSV file and upload it.

Run `make code-gen`.

/cc @horis233

File: deploy/olm-catalog/ibm-mongodb-operator/0.0.1/ibm-mongodb-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -42,37 +42,15 @@ spec:
   customresourcedefinitions:
     owned:
     - description: MongoDB is the Schema for the mongodbs API
-      displayName: MongoDB
```
Comment (horis233) [inline deploy/olm-catalog/ibm-mongodb-operator/0.0.1/ibm-mongodb-operator.v0.0.1.clusterserviceversion.yaml]: We need to keep `displayName:` and `resources:` fields. They are necessary for CSV.

File: deploy/olm-catalog/ibm-mongodb-operator/0.0.1/ibm-mongodb-operator.v0.0.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -42,37 +42,15 @@ spec:
...
       kind: MongoDB
       name: mongodbs.operator.ibm.com
-      resources:
```
Comment (horis233) [inline deploy/olm-catalog/ibm-mongodb-operator/0.0.1/ibm-mongodb-operator.v0.0.1.clusterserviceversion.yaml]: Same as `displayName:`, we need to keep this part

Comment (kgcarr) [reply deploy/olm-catalog/ibm-mongodb-operator/0.0.1/ibm-mongodb-operator.v0.0.1.clusterserviceversion.yaml]: updated

**Comment by kgcarr** [comment]
reverting merge, missed updating the statefulset version to `v1`

**Pull request merged**